types/bootstrap/js/dist/tooltip.d.ts

```diff
@@ -0,0 +1,272 @@
+import * as Popper from 'popper.js';
+
+type PlacementFunction = () => void;
```

Comment:
Same as for my previous comment. Do we need different types for the same signature?

Reply:
I will remove it